clusterloader2/testing/load/config.yaml

```diff
@@ -729,17 +725,11 @@ steps:
 
 - name: Collecting measurements
   measurements:
-  - Identifier: APIResponsiveness
-    Method: APIResponsiveness
-    Params:
-      action: gather
   - Identifier: APIResponsivenessPrometheusSimple
     Method: APIResponsivenessPrometheus
     Params:
       action: gather
-      {{if $ENABLE_PROMETHEUS_API_RESPONSIVENESS}}
```

Comment:
This is currently not enabled for violations in our largest tests (both gce and kubemarks).
I would prefer explicitly enabling it first, and only then removing the rest.

Do we even know if we won't start failing significantly on it?

Reply:
It's enable in Kubemark test (https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml), but you're right - it's not enable in gce5000. I looked through my local repo, where I had unmerged commit. Please see: https://github.com/kubernetes/test-infra/pull/15462